lib/cloudregister/registerutils.py

```diff
@@ -502,6 +502,25 @@ def get_credentials(credentials_file):
     return (username, password)
 
 
+# ----------------------------------------------------------------------------
+def refresh_registry_credentials():
+    """Refresh registry credentials."""
+    # to silence InsecureRequestWarning
+    # should be fixed on a different PR
+    requests.packages.urllib3.disable_warnings()
+    current_server = get_current_smt()
+    if current_server:
+        # system is registered
+        if get_activations():
```

Comment:
the `get_activations()` call already handles update server information processing with the `get_smt()` call as the first executed line in the code https://github.com/SUSE-Enceladus/cloud-regionsrv-client/blob/master/lib/cloudregister/registerutils.py#L409 and the `get_smt()` function makes sure we find an update server.

Meaning the proposed implementation in this PR is sloppy at best, at worst it shows a certain laziness about working to understand the implementation of the utils functions that exist.